xbmc/platform/darwin/DarwinUtils.mm

```diff
@@ -595,6 +604,150 @@ bool CFStringRefToStringWithEncoding(CFStringRef source, std::string &destinatio
   return manufName;
 }
 
+const std::vector<std::string>& CDarwinUtils::GetNameServers(void)
+{
+  static std::vector<std::string> nameServers;
+#if defined(TARGET_DARWIN_IOS)
```

Comment:
mark the whole function `#if defined(TARGET_DARWIN_IOS)`, as it doesn't return anything useful for osx